### ISSUE TITLE
"Segmentation fault" error when trying to run

### ISSUE THREAD

**Issue by bfg01** [atsb/Doom64EX-Plus#26]
It seems I'm cursed regarding this thing...
```
Z_Init: Init Zone Memory Allocator
CON_Init: Init Game Console
G_Init: Setting up game input and commands
M_LoadDefaults: Loading game configuration
WARNING: Unknown Key
I_Init: Setting up machine state.
D_Init: Init DOOM parameters
W_Init: Init WADfiles.
W_AddFile: Adding /home/user/Downloads/Doom64EX-Plus/doom64ex-plus.wad
M_Init: Init miscellaneous info.
R_Init: Init DOOM refresh daemon.
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: cHRM chunk does not match sRGB
libpng warning: Interlace handling should be turned on when using png_read_image
P_Init: Init Playloop state.
NET_Init: Init network subsystem.
S_Init: Setting up sound.
fluidsynth: warning: SDL2 not initialized, SDL2 audio driver won't be usable
fluidsynth: error: Failed to create PulseAudio connection
WARNING: I_InitSequencer: failed to create audio driver
D_CheckNetGame: Checking network game status.
ST_Init: Init status bar.
GL_Init: Init OpenGL
GL_VENDOR: Intel Open Source Technology Center
GL_RENDERER: Mesa DRI Intel(R) HD Graphics 4000 (IVB GT2)
GL_VERSION: 3.0 Mesa 20.3.5
GL_MAX_TEXTURE_SIZE: 16384
GL_MAX_TEXTURE_UNITS_ARB: 8
GL Extension: GL_ARB_multitexture = true
GL Extension: GL_EXT_compiled_vertex_array = true
GL Extension: GL_ARB_texture_env_combine = true
GL Extension: GL_EXT_texture_env_combine = true
GL Extension: GL_EXT_texture_filter_anisotropic = true
Segmentation fault
```

I have libfluidsynth2 2.1.7 installed; I don't have pulseaudio installed but apulse instead.
Inxi info about GPU:
```
Display: server: X.Org v: 1.20.11 driver: X: loaded: modesetting
  unloaded: fbdev,vesa gpu: i915 resolution: 1366x768~60Hz
OpenGL: renderer: Mesa DRI Intel HD Graphics 4000 (IVB GT2)
  v: 4.2 Mesa 20.3.5 direct render: Yes
```

EDIT:
Installed PulseAudio; now instead of this part:
```
fluidsynth: error: Failed to create PulseAudio connection
WARNING: I_InitSequencer: failed to create audio driver
```

I get this one:
```
fluidsynth: Using PulseAudio driver
WARNING: CVar s_soundfont doesn't point to a file.
Found SoundFont /home/user/Downloads/Doom64EX-Plus/doomsnd.sf2
fluidsynth: warning: Failed to set thread to high priority
fluidsynth: Sample 'SFX_008' has no flags set, assuming mono
fluidsynth: Sample 'SFX_022' has no flags set, assuming mono
[...]
fluidsynth: Sample 'SFX_123' has no flags set, assuming mono
fluidsynth: warning: No preset found on channel 9 [bank=128 prog=0]
```

What's wrong with this setup?
Thanks yet again beforehand.

**Comment by atsb** [comment]
Hmmm this segfault could be the vsync setting, I tested it on my Linux box (HD620) and it was ok.  What you can try is, in gl_main.c there will be a call to SDL_GL_SetSwapInterval.  Comment out this function call and see if that works for you.

**Comment by bfg01** [comment]
Sorry, no dice; exact same errors...

**Comment by bfg01** [comment]
Finally solved on #29 . Let's look forward a "stable" milestone of the project in the future. Thanks.
